.github/workflows/documentation_gen.yaml

```diff
@@ -59,7 +59,9 @@ jobs:
     - name: Install packages needed for build
       run: sudo apt-get install libsasl2-dev gcc
     - name: Install wkhtmltopdf
-      run: sudo apt-get install wkhtmltopdf
+      run: |-
+        sudo apt-get update
+        sudo apt-get install wkhtmltopdf
```

Comment:
is there a need to run update?

Reply:
@YS-L and i were seeing some errors that prompted us to add this. can remove if not needed.